doc/quickstart.md

```diff
@@ -1,22 +1,47 @@
 # Quickstart
 
-## Dependencies
+## Requirements
 
 - make
 - kubectl
-- clouds.yaml
-- environment.tfvars
+- terraform
+- yq v2
```

Comment:
For me, it works only with v4.

Reply:
Maybe add a note that there are two variants of `yq` which are in principle both supported, but the version based detection needing improvement?

Reply:
Good hint. Pip installed `yq 3.1.1` and I just thought that's the newest so I downgraded to the latest `v2` and it worked